contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql

```diff
@@ -2543,6 +2543,241 @@ select
 from sys.objects s;
 GRANT SELECT ON sys.sysobjects TO PUBLIC;
 
+-- internal function in order to workaround BABEL-1597
+CREATE OR REPLACE FUNCTION sys.columns_internal()
```

Comment:
Please add upgrade tests for all updated functions/tables view. We should test when these objects are accesses from views/functions/procedures

Reply:
Yes, I had just created this PR to be able to run the MVU tests to check upgrade script compatibility